### ISSUE TITLE
A modern version gives you access to the best plugins

### ISSUE THREAD

**Issue by felixarntz** [wp-core-php/servehappy#4]
Several popular quality plugins require a PHP version higher than 5.2. When using an outdated version, you do not have access to many of these solid plugins. Therefore the experience will be much more seamless on a supported version.

**Comment by JDGrimes** [comment]
Might be interesting to look at the top X plugins and see how many of them require a more recent version.

**Comment by Zodiac1978** [comment]
Here is a list of the top 120 plugins: http://wp-info.org/plugin-top120/

**Comment by mindctrl** [comment]
I don't think the wp-info.org top 120 list is accurate. For example, it's missing WP Forms, which has 500k+ active installs. https://wordpress.org/plugins/wpforms-lite/

**Comment by JDGrimes** [comment]
Also worth noting that some plugins recommend higher versions, even if they don't require them. [WooCommerce recommends PHP 5.6+](https://docs.woocommerce.com/document/server-requirements/). And of course, [Yoast SEO is actively pushing users to update](https://wptavern.com/yoast-seo-4-5-urges-users-to-upgrade-to-php-7). These plugins are both in the top 10.

**Comment by Zodiac1978** [comment]
@JDGrimes Yes, that's true. BackWPup is working with PHP 5.2, but then it does not support all backup methods.

@mindctrl Nice catch. I ping the author of the page.

**Comment by ePascalC** [comment]
@mindctrl @Zodiac1978 The job to bring in the plugins from the top120 seems indeed to have failed for weeks. It's now updated and will run regularly again. Thanks for notifying!
